src/v0/util/errorTypes/networkError.js

```diff
@@ -0,0 +1,36 @@
+const tags = require("../tags");
+const { BaseError } = require("./base");
+
+const errorTypes = Object.values(tags.ERROR_TYPES);
+const metaTypes = Object.values(tags.METADATA);
+
+class NetworkError extends BaseError {
```

Comment:
Should we rename this class to `NetworkAbortedError` to avoid misuse for any kind of network related errors ?

Reply:
There is `AbortedError` for the same purpose. This class can be helpful to throw errors when you have to dynamically deduce the error type (abort, retry or throttle) and set the `errorType` stat tag.

Reply:
Then let's not set the default error type to ABORTED - Ln 17

Reply:
I see default status is set to 400, so aborted. Should be fine as they can override.